edit-post/components/header/style.scss

```diff
@@ -11,12 +11,11 @@
 	left: 0;
 	right: 0;
 
-	// mobile edgecase for toolbar
+	// Stick the mobile toolbar way to the top, because the adminbar is not fixed either.
```

Comment:
What does "way to the top" mean? I think it's a turn of phrase but I don't really understand 🤷‍♂️

Reply:
Also I think "adminbar" should be "admin bar".

Reply:
Good catches, both of them.